src/Kunstmaan/MultiDomainBundle/Router/DomainBasedLocaleRouter.php

```diff
@@ -9,23 +9,12 @@
 use Symfony\Component\Routing\Generator\UrlGenerator;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\Matcher\UrlMatcher;
-use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 
 class DomainBasedLocaleRouter extends SlugRouter
 {
-    /** @var RouteCollection */
-    protected $routeCollectionMultiLanguage;
-
-    /**
-     * @var array|null
-     */
-    private $otherSite;
-
-    /**
-     * @var array
-     */
-    private $cachedNodeTranslations = [];
+    private ?array $otherSite = null;
```

Comment:
The switch from protected to private properties is a BC break and should be reverted

Reply:
This property was already private